module/Endjin.ZeroFailed.Build.psm1

```diff
@@ -0,0 +1,16 @@
+# <copyright file="Endjin.ZeroFailed.Build.psm1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+
+# find all the functions that make-up this module
+$functions = Get-ChildItem -Recurse $PSScriptRoot/functions -Include *.ps1 | `
+                                Where-Object { $_ -notmatch ".Tests.ps1" }
+                    
+# dot source the individual scripts that make-up this module
+foreach ($function in ($functions)) { . $function.FullName }
+
+# export the non-private functions (by convention, private function scripts must begin with an '_' character)

```

Comment:
Is this redundant?

Reply:
We treat this as boilerplate in case we add exportable functions in the future.